cuckoo/finalizers.py

```
@@ -61,6 +63,20 @@ def __init__(
         super().__init__()
         self._node: BinaryTreeNode = node
 
+    def _resolve_column_selection(
+        self,
+        columns: Optional[TCOLUMNS] = None,
+        invert_selection=False,
+    ):
+        if columns is None:
+            columns = DEFAULT_COLUMNS_INVERTED if invert_selection else DEFAULT_COLUMNS
+        if invert_selection:
+            columns_sanatized = {col for col in columns if isinstance(col, str)}
```

if you invert the selection, `Include`s are invalid items in the `columns` list and are silently ignored.

---

I would prefer if an error was raised, instead of ignoring `Include` statements. 

Also, how does this work if I pass in `columns=["author { name, address }"]` instead of using an `Include`?

---

Ya I use the columns as a string hack sometimes

---

good point! will raise an error instead